tests/queries/0_stateless/02402_merge_engine_with_view.reference

```diff
@@ -0,0 +1,6 @@
+2
```

Comment:
does something guarantee this particular order of numbers in output? if no, pls add `order by` to your query

Reply:
I will add it